Eask

```diff
@@ -0,0 +1,68 @@
+;; Eask file for project Cody.
+(source "gnu")
+(source "melpa")
+
+(package "cody"
+         "0.2.0"
+         "Sourcegraph Cody in Emacs")
+
+(website-url "https://github.com/sourcegraph/emacs-cody")
+(keywords "completion" "convenience" "languages" "programming" "tools")
+
+(package-file "cody.el")
+
+(depends-on "emacs" "26.3")
+(depends-on "jsonrpc" "1.0.16")
+(depends-on "uuidgen" "20240201.2318")
+
+(development
+ (depends-on "buttercup" "1.35"))
+
+(put 'script 'lisp-indent-function '1)
+(script "test"
+  (require 'package)
+  (add-to-list 'package-archives '("melpa" . "https://melpa.org/packages/") t)
+
+  (package-initialize)
+
+  (let ((bootstrap-file (expand-file-name "straight/repos/straight.el/bootstrap.el" user-emacs-directory))
+        (bootstrap-version 5))
+    (unless (file-exists-p bootstrap-file)
+      (with-current-buffer
+          (url-retrieve-synchronously
+           "https://raw.githubusercontent.com/raxod502/straight.el/develop/install.el"
+           'silent 'inhibit-cookies)
+        (goto-char (point-max))
+        (eval-print-last-sexp)))
+    (load bootstrap-file nil 'nomessage))
```

Comment:
FYI when you next want to yakshave your init.el, latest versions of emacs have pretty decent builtin package management which also support installing from git repo like straight does. I recently switched from straight to just using that. https://github.com/keegancsmith/dotfiles/commit/09833c0e2ac306bce472dfead7c095c213ef476c